deploy/charts/kiali-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
```

Comment:
Same comment here regarding indentation. Keep it looking at identical as possible to the other deployment.yaml.

That way we can do a diff and determine what, if anything, changed between one versus the other (this lets us more easily keep them in sync).